### ISSUE TITLE
Clips Adjustment is not Accessible via Keyboard

### ISSUE THREAD

**Issue by chinakov** [audacity/audacity#10985]
### Bug description

If the User navigates to any given clip via keyboard, the User won't be able to crop or stretch clips (or access context menu). 

Using arrow keys selects other clips, tabbing goes to next UI element. Enter disables and enables the selection. 

### Steps to reproduce

1. Launch AU4
2. Import Clip
3. Navigate via accessibility to the clip
4. Try to trim, stretch or access context menu (or rename) 
5. Observe result

### Expected behavior

The User should be able to trim, stretch, access context menu or rename a clip

### Audacity Version

beta

### Operating system

OS independent

### Additional context

https://github.com/user-attachments/assets/8a5f4d07-8619-477b-96d7-b5082c0c840d

**Comment by DilsonsPickles** [comment]
@chinakov this should be solved with keyboard shortcuts. 

Shift + cmd/ctrl + arrows for trimming
Shift + option/alt + arrows for stretching
Shift + f10 for opening context menu

I don't think we have shortcuts for stretching in yet though
